modules/ndu/src/backend/training/model-service.ts

```diff
@@ -0,0 +1,92 @@
+import * as sdk from 'botpress/sdk'
+import crypto from 'crypto'
+import fse, { WriteStream } from 'fs-extra'
+import _ from 'lodash'
+import path from 'path'
+import { Stream } from 'stream'
+import tar from 'tar'
+import tmp from 'tmp'
+
+export interface Model {
+  hash: string
+  startedAt: Date
+  finishedAt: Date
+  success: boolean
+  data: any
+}
+
+export const MODELS_DIR = './models'
+const MAX_MODELS_TO_KEEP = 2
```

Comment:
IMO 1 is enough :p let's always keep the latest version of the model.
The reason we keep 2 in NLU is basically that nlu is kindof long to train .. and if the latest one is for some reason corrupted, we don't want to retrain the whole thing, load the other previous. in your case it trains very quickly so we don't need an history

Reply:
Good point, I tried to keep the same logic as NLU, in case we make a service to handel models, but I can limit it to 1